packages/transition-backend/src/models/db/transitScenarios.db.queries.ts

```diff
@@ -112,192 +114,256 @@ const read = async (id: string) => {
     }
 };
 
-const create = async (newObject: ScenarioAttributes, returning?: string) => {
+const _create = async (
```

Comment:
Please add a comment to explain why we have a private create

Reply:
(It probably need a more elaborate name than juste _create

Reply:
Unless the private functions are reused by multiple public ones or are really huge (like in the schedules), I'll nest them in their public counterpart to limit their scope. Like what is done in the default.db.queries `updateMultiple` function.